### PR TITLE
feat(uidl-schema): uidl-schema-update

### PR DESCRIPTION
**Pull request by anamariaoros**

### CODE REVIEW

File: src/uidl-definitions/schemas/new-component.json [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+                    }
+                  }
+                },
+                "defaultValue": {},
+                "actions": {"type": "array"}
+              }
+            }
+          }
+        },
+        "style": {},
```
Comment (alexnm) [inline src/uidl-definitions/schemas/new-component.json]: just a quick note, style could also be now expanded into `<any-key>`: staticValue / dynamicReference + nested styles, you can talk with @vladnicula about the structure tomorrow

File: src/uidl-definitions/schemas/new-component.json [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+        "node": {
+          "oneOf": [{
```
Comment (alexnm) [inline src/uidl-definitions/schemas/new-component.json]: this oneOf is good, but we could also add a separate `string` as a type, if we want to support the old style text nodes, ex: "$props.title". let's talk tomorrow

File: src/uidl-definitions/schemas/new-component.json [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+          "type": "object",
+          "additionalProperties": false,
+          "required": [ "type", "content" ],
+          "properties": {
+            "type": {
+              "type": "string",
+              "enum":  ["repeat"]
+            },
+            "content": {
+              "$ref": "#/definitions/repeatContent"
```
Comment (alexnm) [inline src/uidl-definitions/schemas/new-component.json]: no need to split this if it's not reused somewhere else

Comment (vladnicula) [reply src/uidl-definitions/schemas/new-component.json]: I'd argue we could split it for ease of editing. I'd rather have the content of the repeat in a separate file for example and only focus on it if we make changes in the future.

**Comment by vladnicula** [comment]
Will need this merged before integrating #118

**Pull request merged**